docs/sources/flow/tasks/send-profiles-to-pyroscope/index.md

```diff
@@ -0,0 +1,133 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/tasks/send-profiles-to-pyroscope/
+canonical: https://grafana.com/docs/agent/latest/flow/send-profiles-to-pyroscope/
+description: Send profiles to Pyroscope 
+title: Send profiles Pyroscope 
+weight: 120
+---
+
+# Send profiles to Pyroscope
+
+Learn how to configure {{< param "PRODUCT_NAME" >}} to collect [Pyroscope Server][] profiles and forward them to a [Pyroscope Server][].
+
+This topic describes how to:
+
+* Setup [Pyroscope Server][].
+* Setup {{< param "PRODUCT_NAME" >}} to collect profiles and send them to [Pyroscope Server][].
+
+## Components used in this topic
+
+* [pyroscope.write][]
+* [pyroscope.scrape][]
+
+## Before you begin
+
+* Download {{< param "PRODUCT_NAME" >}}.
+* Install [docker][] if it is not already installed.
```

Comment:
```suggestion
* Install [docker][].
```

Is it safe to assume that Grafana Agent needs to be installed to do anything with Agent? and we don't need to mention it?

Also saying "Download Grafana Agent... then what? We either need to be explicit and state that they need to Install it or not mention it (going with it's implied).